src/core/main.cpp

```diff
@@ -494,8 +495,25 @@ int main( int argc, char * * argv )
 		}
 		else
 		{
-			Engine::getSong()->createNewProject();
-
+			// If enabled, open last project if there is one. Else, create
+			// a new one.
+			if( !ConfigManager::inst()->value( "app", "disableopenlast" ).toInt() )
+			{
+				if( !ConfigManager::inst()->recentlyOpenedProjects().isEmpty() )
+				{
+					Engine::getSong()->loadProject(
+									ConfigManager::inst()->
+										recentlyOpenedProjects().first() );
+				}
+				else
+				{
+					Engine::getSong()->createNewProject();
+				}
+			}
+			else
+			{
+				Engine::getSong()->createNewProject();
+			}
 			// [Settel] workaround: showMaximized() doesn't work with
```

Comment:
*If you want*, the above could be changed to something like: 
```c++
			// If enabled, open last project if there is one. Else, create
			// a new one.
			if( !ConfigManager::inst()->value( "app", "disableopenlast" ).toInt() && 
				!ConfigManager::inst()->recentlyOpenedProjects().isEmpty()) )
			{
				Engine::getSong()->loadProject(
								ConfigManager::inst()->
									recentlyOpenedProjects().first() );
			}
			else
			{
				Engine::getSong()->createNewProject();
			}
```
to avoid the repeat call to `Engine::getSong()->createNewProject()`.